### DISCUSSION TITLE
grayscale filter not working / unused --tw-grayscale custom prop

### DISCUSSION THREAD

**Discussion by nachtfunke** [tailwindlabs/tailwindcss#7314] [Help]
Hello!

I noticed that the grayscale isn't working because the class only applies a custom property but doesn't actually set the `filter` prop. Can someone explain to me how the custom prop is supposed tobe applied normally?

**Comment by kerwin-wildsea** [comment]
I found this just before I found the solution. You need to add both `filter grayscale`. It's something the docs aren't specifically clear about. 

The v2 docs on the other hand are very clear about it: https://v2.tailwindcss.com/docs/grayscale#usage

> Use the grayscale and grayscale-0 utilities alongside the filter utility to whether an element should be rendered as grayscale or in full color.
> 
> ```html
> <div class="filter grayscale ..."> 
>  
> </div> 
> ```
